app/scripts/directives/bindService.js

```diff
@@ -16,10 +17,16 @@ angular.module('openshiftConsole').component('bindService', {
 });
 
 function BindService($filter,
+                     BindingModalUtils,
                      DataService,
                      DNS1123_SUBDOMAIN_VALIDATION) {
   var ctrl = this;
 
```

Comment:
@jwforres thoughts here, pulled some things from `bindService` to share w/both components.